CHANGELOG.rst

```diff
@@ -1,6 +1,32 @@
 Changelog
 =========
 
+1.12.0 (2022-12-02)
+-------------------
+
+Feature
+"""""""
+
+- Add custom XCom backend for GCS.
+  (`#681 <https://github.com/astronomer/astronomer-providers/pull/681>`_)
+
+Bug Fixes
+"""""""""
+
+- Change return value of ``SnowflakeOperatorAsync`` to be same as the Sync operator.
+  (`#781 <https://github.com/astronomer/astronomer-providers/pull/781>`_)
+- Add poke_interval to ``S3KeySensorAsync``.
+  (`#782 <https://github.com/astronomer/astronomer-providers/pull/782>`_)
+- Fix ``AwsBaseHookAsync`` to get connection details using ``AwsConnectionWrapper``.
```

Comment:
this sentence need to be from a user point of view

Reply:
Use ``AwsConnectionWrapper`` to get connection details to create AWS async client.